README.md

```diff
@@ -4,109 +4,71 @@
     </a>
 </p>
 
-<h1 align="center">Plugin Skeleton</h1>
+## Features
 
-<p align="center">Skeleton for starting Sylius plugins.</p>
+## Requirements
 
-## Documentation
+| | Version |
+| :--- | :--- |
+| PHP  | 7.3+ |
```

Comment:
This makes me think. Wouldn't we limit version to PHP 7.4 min (to have strong typing)?
WDYT ?

Reply:
I agree for the strong typing, but we want the plugin to be able on sylius 1.8 too, sylius 1.8 need "php": "^7.3"... So i think we should go for PHP 7.3 + for the first release

Reply:
I think we can do that on another PR, but yes, we need PHP 7.4 mini.